Dockerfile

```
@@ -0,0 +1,24 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+ARG BUILD_CONFIGURATION=Release
+WORKDIR /src
+COPY ["Paper-API/Paper.csproj", "Paper-API/"]
```

```suggestion
COPY ["Paper-API/", "Paper-API/Paper.csproj"]
```

For some reason, you have to copy `Paper-API` before you're able to access `Paper-API/Paper.csproj` 🤷